packages/create-sitecore-jss/src/templates/vue/data/routes/styleguide/{{language}}.yml

```diff
@@ -0,0 +1,26 @@
+fields:
```

Comment:
This file has changed, see NextjsStyleguide or Angular initializer for guidence.